### DISCUSSION TITLE
Use with non-Voron printer

### DISCUSSION THREAD

**Discussion by Avpman2** [Enraged-Rabbit-Community/ERCF_v2#99] [Q&A]
Would this work on a Flashforge Adventurer 5 running the Klipper mod? I saw posts about a control board, but don't know if I need that. I believe there is an extra USB port on the Flashforge board.

**Comment by mnorris-CAI** [comment]
ERCF is designed to be printer agnostic, so, in theory at least, it will work with _any_ printer

If you want ERCF with all the bells and whistles, such as filament sensor per gate, NeoPixels, etc., I'd recommend looking into a dedicated ERCF MCU, such as the BTT MMB

**Reply by Avpman2** [reply]
I saw ads for that board but no info on how it hooks up to the printer and what it provides. Might you have a link for detailed info on the BTT MMB? I've seen them on Amazon, but so far, everywhere references it's for  Voron printers.

**Reply by mnorris-CAI** [reply]
The best source of info is the [BigTreeTech MMB GitHub](https://github.com/bigtreetech/MMB)

**Reply by TurboTronix** [reply]
What about the erf, is it compatible with any hotend? Can't figure out if it'll mount on my e3d volcano hotend on my Ender 3.

**Reply by mnorris-CAI** [reply]
Yep, _should_ work with _any_ hotend

**Comment by SPYKE357** [comment]
Hi all does anyone know if/how the ERCF v2 can be wired to the Nebula pad on a Ender 3 V3 KE, running rooted klipper?

**Reply by norris108** [reply]
As long as you have a spare USB port to connect to your ERCF MCU and it's running mainline Klipper, _not_ a Creality butchered version of Klipper, then, in theory, it _should_ work fine

**Comment by JSochaJR** [comment]
Hello I'm new here does anyone know if you can make the ERFC work with the artillery X4 Plus since it comes with klipper on it?

**Reply by norris108** [reply]
As long as you have a spare USB port to connect to your ERCF MCU and it's running mainline Klipper, not an Artillery butchered version of Klipper, then, in theory, it should work fine
